charts/values.yaml

```
@@ -17,15 +17,13 @@ cloudprovider: "digitalocean"
 
 kubernetesVersion: ""
 
-# general RKE options
-rke:
-  # enable local auth endpoint
-  localClusterAuthEndpoint:
-    enabled: false
-  # specify fqdn of local access endpoint
-  # fqdn: foo.bar.example
-  # specify cacert of local access endpoint
-  # caCerts: ""
+# enable local auth endpoint
+localClusterAuthEndpoint:
+  enabled: false
+#  specify fqdn of local access endpoint
+#  fqdn: foo.bar.example
+#  specify cacert of local access endpoint
+#  caCerts: ""
```

```suggestion
  enabled: false
  # specify fqdn of local access endpoint
  # fqdn: foo.bar.example
  # specify cacert of local access endpoint
  # caCerts: ""
```
This keeps the commenting structure in line with the rest of the file, and is the expected format when using a text editor to uncomment.